0x48piraj/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
Although! you had written awesome code but if someone newbie wants to understand your code than he will not able to so please use proper indentaion. If you use atom you can use [atom-beautify](https://atom.io/packages/atom-beautify)

Reply:
Indented the code. :+1: